ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,21 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: console-test
+      cli: latest
+      commands:
+        frontend/test-frontend.sh
+      from: bin
+      resources:
+        requests:
+          cpu: "2"
+          memory: 8Gi
```

Comment:
@yapei sorry for miss this comment.
I do not think 8Gi and 2 cpu are needed for it because your code does not run golang.
please try to reduce resource. thanks

Reply:
@kuiwang02  web browser also use lots of memory, I can reduce to `4Gi` to see if it is broken, HDYT?

Reply:
if you ever observe the memory usage, take that value. if not, I am OK to take 4Gi.